src/main/java/com/epam/machine/controller/AdminCertainOrderServlet.java

```diff
@@ -5,17 +5,62 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.io.IOException;
+import java.sql.SQLException;
+import java.util.List;
+
+import com.epam.machine.entity.Offer;
+import com.epam.machine.service.OfferServiceImpl;
 import org.apache.log4j.Logger;
 
 public class AdminCertainOrderServlet extends HttpServlet {
     final static Logger logger = Logger.getLogger(AdminCertainOrderServlet.class);
+    private OfferServiceImpl offerService = new OfferServiceImpl();
+    private int offerId;
+    private List<Offer> offers;
 
     @Override
     public void doGet (HttpServletRequest request, HttpServletResponse response) {
         try {
+            offerId = Integer.parseInt(request.getParameter("val"));
+            offers = offerService.getAll();
+
             request.getRequestDispatcher("/adminCertainOrder.jsp").forward(request, response);
-        } catch(ServletException | IOException err) {
+        } catch(ServletException | IOException | SQLException | ClassNotFoundException err) {
             logger.error("Something is wrong. Game over. Try again. " + err.getMessage());
         }
     }
+
+    @Override
+    protected void doPost(HttpServletRequest request, HttpServletResponse response) {
+        int statusId = Integer.parseInt(request.getParameter("status"));
```

Comment:
what if there's no status passed, or text, or it bigger then 4?
there should be validation for these cases